rnns/fastrnns/custom_lstms.py

```diff
@@ -246,12 +252,20 @@ def init_stacked_lstm(num_layers, layer, first_layer_args, other_layer_args):
 
 
 class StackedLSTM(jit.ScriptModule):
-    __constants__ = ['layers']  # Necessary for iterating through self.layers
+    # Necessary for iterating through self.layers and dropout support
+    __constants__ = ['layers', 'num_layers', 'dropout']
 
-    def __init__(self, num_layers, layer, first_layer_args, other_layer_args):
+    def __init__(self, num_layers, layer, first_layer_args, other_layer_args,
+                 dropout=0.0):
         super(StackedLSTM, self).__init__()
         self.layers = init_stacked_lstm(num_layers, layer, first_layer_args,
                                         other_layer_args)
+        # If non-zero, introduces a Dropout layer on the outputs of each LSTM
+        # layer except the last layer, with dropout probability = dropout.
+        self.dropout = dropout
+        self.num_layers = num_layers
+        self.dropout_layer = jit.trace(nn.Dropout(dropout), torch.randn(2, 2),
```

Comment:
Is the tracing necessary -- can't we just do `self.dropout_layer = nn.Dropout(dropout)`?

Reply:
Ohh you are right, nn.Dropout is a weak script module so it should be automatically converted to a ScriptModule here, I will update it.